packages/toolpad-core/src/AppProvider/AppProvider.tsx

```diff
@@ -191,7 +202,7 @@ AppProvider.propTypes /* remove-proptypes */ = {
   }),
   /**
    * [Theme or themes](https://mui.com/toolpad/core/react-app-provider/#theming) to be used by the app in light/dark mode. A [CSS variables theme](https://mui.com/material-ui/experimental-api/css-theme-variables/overview/) is recommended.
-   * @default extendTheme()
+   * @default createTheme()
```

Comment:
```suggestion
   * @default createTheme({ cssVariables: true, colorSchemes: { light: true, dark: true } })
```

The CSS variables feature is not enabled by default.